source/_posts/2017/2017-06-17-php-object-calisthenics-rules-made-simple-version-3-0-is-out-now.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "PHP Object Calisthenics Made Simple - Version 3.0 is Out Now"
+perex: '''
+    Object Calisthenics are 9 language-agnostic rules to help you write better and cleaner code.
+    They help you to get rid of else, method chaining, long classes or functions, unreadable short names and so on.
+    <br><br>
+    <strong>Object Calisthenics 3.0 runs on CodeSniffer 3.0, PHP 7.1 and brings 6 of them with fancy configuration and code examples</strong>.
+'''
+lang: en
+---
+
+If you are coding standard nerd like me, you'll probably more than PSR2 standards in your ruleset. But even if you fon't, [Object Calisthenics](https://github.com/object-calisthenics/phpcs-calisthenics-rules) is developer-friendly game changer for your code.
```

Comment:
- you are **a** coding standard nerd
 - probably **have** more ... than **just PSR-2** standards
 - fon't -> don't
- ... is **a** developer-friendly